package.json

```diff
@@ -14,8 +14,9 @@
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.1",
+    "socket.io-client": "^4.1.2",
     "styled-components": "^5.3.0",
-    "typescript": "^4.1.3",
+    "typescript": "^4.3.2",
```

Comment:
Any particular reason to update the TypeScript version?

Reply:
The editor was showing an error while trying to make it a UMD module by using `export as namespace "synced-video-player";`.
Upgrading the package solved it partially. But then there was this new error that appeared after and I removed the above line for UMD.

I don't think upgrading it would be a problem?

Reply:
Upgrading core infra should be done with caution
Can you check if you still need the new version?

Reply:
No. Downgraded to 4.1.2.